src/default_authorized_user.rs

```diff
@@ -114,8 +106,6 @@ struct UserCredentials {
     pub(crate) quota_project_id: Option<String>,
     /// Refresh Token
     pub(crate) refresh_token: String,
-    /// Type
-    pub(crate) r#type: String,
```

Comment:
I have to remove the `type` field because serde strips it when using `#[serde(tag = "type")]`, and this is not used anywhere in the crate or exposed